identity-aware-proxy/IAPClientTest/IAPTokenVerificationTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) 2020 Google LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+// 
+// https://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+using Google.Apis.Auth;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace IAP.Samples.Tests
+{
+    [Collection(nameof(IAPFixture))]
+    public class IAPTokenVerificationTests
+    {
+        private readonly IAPFixture _fixture;
+
+        public IAPTokenVerificationTests(IAPFixture fixture) => _fixture = fixture;
+
+        [Fact]
+        public async Task Verifies()
+        {
+            // Let's use our IAPClient, so we make sure that we can actually verify a valid IAP token.
+            IAPClient client = new IAPClient();
+            HttpResponseMessage response = await client.InvokeRequestAsync(
+                _fixture.IapClientId, _fixture.CredentialsFilePath, _fixture.IapUrl);
+            string bouncedToken = await response.Content.ReadAsStringAsync();
```

Comment:
What does "bounced" mean here?

Reply:
The application returns the IAP token back, i.e., bounces the token back. I'll add a comment explaining.